--- go/nativeMessagingHost.go	2025-09-02 00:37:04
+++ nightly/go/nativeMessagingHost.go	2025-09-02 15:48:24
@@ -186,7 +186,7 @@
 // MacOS and Linux: returns a list of all open ports for all instances of anytype found using cli utilities lsof and grep
 func getOpenPortsUnix() (map[string][]string, error) {
 	// execute the command
-	appName := "anytype"
+	appName := "anytype-dev"
 	stdout, err := execCommand(`lsof -i -P -n | grep LISTEN | grep "` + appName + `"`)
 	Trace.Print(`lsof -i -P -n | grep LISTEN | grep "` + appName + `"`)
 	if err != nil {
